sdk/react/src/views/AdPlaybackScreen/AdPlaybackScreen.js

```diff
@@ -244,13 +246,22 @@ export default class AdPlaybackScreen extends Component {
     return iconViews;
   };
 
-  render() {
+  componentDidUpdate(prevProps: Readonly<P>, prevState: Readonly<S>, snapshot: SS) {
```

Comment:
where do these types come from?

Reply:
It auto-generated code, if you can suggest me how write this method correctly i would be glad.

Reply:
this file is not covered with Flow, no types needed

Reply:
deleted, thanks!